payments-core/src/test/java/com/stripe/android/model/parsers/PaymentMethodMessageJsonParserTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.model.parsers
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.PaymentMethodMessageFixtures
+import org.json.JSONObject
+import org.junit.Test
+
+class PaymentMethodMessageJsonParserTest {
```

Comment:
Should we add a case for the error case, and what’s the desired behavior for that?